app/models/caution_flag.rb

```diff
@@ -30,14 +30,18 @@ def self.cols_to_update(cols_map_update)
   end
 
   def self.apply_update(rule, ids)
-    cols_map_update = %i[
-      title description link_text link_url
-    ]
-
     if rule.link_url == CautionFlagRule::SCHOOL_URL
-      cols_map_update.pop
+      cols_map_update = %i[
```

Comment:
This method is now failing the linting with these additional lines.  I recommend braking the two branches of the `if` into their own method.  Then a test could be added for each of the methods.  So this method would look something 
like this:
```
def self.apply_update(rule, ids)
    apply_update_sql = 
      if rule.link_url == CautionFlagRule::SCHOOL_URL
        school_url_update_sql(rule, ids)
      else
        basic_update_sql(rule, ids)
      end

    CautionFlag.connection.update(apply_update_sql)
  end

  def self.school_url_update_sql
    [...]
  end

  def self.basic_update_sql
    [...]
  end
```

Reply:
codeclimate linting does not equal rubocop linting within the repository, this branch passes `make ci`

Reply:
See https://github.com/department-of-veterans-affairs/gibct-data-service/blob/master/.rubocop.yml#L90

Reply:
pulled out school url specific sql to its own method